src/routes/ap.rs

```diff
@@ -1,51 +1,22 @@
 use itertools::Itertools;
```

Comment:
are all other routes than `ui` backend routes, technically?

Reply:
the tl;dr is that everything that's not `ui` is federation

Reply:
oh ok

yea it would be handy to have a line of module-level doc (`//!') to tip the reader off as to what the expected area of responsibility for the module is